src/components/Eth2list.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
```

Comment:
I'd prefer if this component didn't have to exist. Instead, could we pass the `articles` object to a generic list component? I'll think on how to do this, perhaps via frontmatter of the markdown file?

Reply:
I'd love that but I just wasn't sure how to do it in the markdown.